VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -35,6 +35,10 @@
  * This class initializes the modules of the app based on the configuration.
  */
 public class MainApplication extends Application{
+    //FIXME - temporary solution
+    public static final boolean Q4_ASSESSMENT_FLAG = true;
```

Comment:
Maybe a better place to put this would be ``Config``? That way it goes with all the other runtime options.

Also, these should be ``false`` on master until we're ready to turn it on.